src/index.js

```diff
@@ -37,6 +37,7 @@ const oidcConfig = {
   client_id: GlobalConfiguration.AWS_CLIENT_ID,
   redirect_uri: GlobalConfiguration.OAUTH2_REDIRECT_URL,
   userStore: new WebStorageStateStore({ store: window.localStorage }),
+  automaticSilentRenew: true,
```

Comment:
This was already the default but I added it explicitly so we wouldn't have to dig through the docs to determine the setting